### ISSUE TITLE
Package dependency of gstatus

### ISSUE THREAD

**Issue by sankarshanmukhopadhyay** [Tendrl/gluster-integration#151]
From @japplewhite 

```
I have been testing and now on my storage nodes when I try to update I get:

[root@gluster-02 tendrl]# yum update -y
Loaded plugins: fastestmirror
centos-gluster39
                                                                          |
2.9 kB  00:00:00
Loading mirror speeds from cached hostfile
 * base: mirror.redsox.cc
 * epel: fedora-epel.mirrors.tds.net
 * extras: mirror.cogentco.com
 * updates: mirrors.centos.webair.com
Resolving Dependencies
--> Running transaction check
---> Package tendrl-gluster-integration.noarch 0:1.2-02_23_2017_13_46_10
will be updated
---> Package tendrl-gluster-integration.noarch 0:1.2-02_23_2017_20_10_12
will be an update
--> Processing Dependency: gstatus for package:
tendrl-gluster-integration-1.2-02_23_2017_20_10_12.noarch
---> Package tendrl-node-agent.noarch 0:1.2-02_23_2017_15_56_11 will be
updated
---> Package tendrl-node-agent.noarch 0:1.2-02_24_2017_00_01_02 will be an
update
--> Finished Dependency Resolution
Error: Package: tendrl-gluster-integration-1.2-02_23_2017_20_10_12.noarch
(tendrl-tendrl)
           Requires: gstatus
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by sankarshanmukhopadhyay** [comment]
While the project addresses the Requires issue in the packaging spec, a work-around to check could be using the gstatus package available from https://github.com/gluster/gstatus/blob/master/rpms/gstatus-0.64-3.el7.x86_64.rpm

**Comment by anmolsachan** [comment]
While performing installation from source, I am getting the following issue. 

    Processing /root/gluster-integration
      Requirement already satisfied (use --upgrade to upgrade): tendrl-gluster-integration==1.2 from file:///root/gluster-integration in /usr/lib/python2.7/site-packages/tendrl_gluster_integration-1.2-py2.7.egg
    Requirement already satisfied: namespaces in /usr/lib/python2.7/site-packages (from tendrl-gluster-integration==1.2)
    Requirement already satisfied: python-etcd in /usr/lib/python2.7/site-packages (from tendrl-gluster-integration==1.2)
    Requirement already satisfied: python-dateutil in /usr/lib/python2.7/site-packages (from tendrl-gluster-integration==1.2)
    Requirement already satisfied: gevent>=1.0 in /usr/lib64/python2.7/site-packages (from tendrl-gluster-integration==1.2)
    Requirement already satisfied: greenlet>=0.3.2 in /usr/lib64/python2.7/site-packages (from tendrl-gluster-integration==1.2)
    Collecting gstatus (from tendrl-gluster-integration==1.2)
      Could not find a version that satisfies the requirement gstatus (from tendrl-gluster-integration==1.2) (from versions: )
    No matching distribution found for gstatus (from tendrl-gluster-integration==1.2)

I am not sure if a new issue is required for this or not.

**Comment by shtripat** [comment]
We actually need to build gstatus on our own. For the time being I feel you can remove gstatus dependency from requirement.txt and go ahead. Install gstatus from https://github.com/gluster/gstatus/blob/master/rpms/gstatus-0.64-3.el7.x86_64.rpm

**Comment by shtripat** [comment]
@TimothyAsirJeyasing is gstatus build available now and just by adding copr repo, would it get installed?

**Comment by TimothyAsir** [comment]
@shtripat yes, its available from copr repo: https://copr-be.cloud.fedoraproject.org/results/tendrl/tendrl/epel-7-x86_64/00520049-gstatus/

**Comment by shtripat** [comment]
@TimothyAsirJeyasing can you close this issue in that case?

**Comment by fbalak** [comment]
@sankarshanmukhopadhyay This can be closed. Looks ok.
Tested with:
tendrl-gluster-integration-1.2.1-03_08_2017_04_38_03.noarch
tendrl-commons-1.2.1-03_08_2017_04_42_04.noarch
tendrl-node-agent-1.2.1-03_08_2017_00_01_03.noarch

**Comment by nthomas-redhat** [comment]
Closing the issue as it is verified
